### PR TITLE
feat(pie-cookie-banner): DSW-1127 use button for manage preferences link

### PR DESCRIPTION
**Pull request by jamieomaguire**
---
"@justeattakeaway/pie-cookie-banner": minor
---

[Changed] - Use button tag for manage preferences link and update test naming to start with 'should'

### CODE REVIEW

File: packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ test.describe('PieCookieBanner - Component tests', () => {
         expect(cookieBanner).toBeVisible();
     });
 
-    test('clicking "Accept all" emits the correct event and closes the cookie banner', async ({ mount, page }) => {
+    test('should clicking "Accept all" emits the correct event and closes the cookie banner', async ({ mount, page }) => {
```
Comment (kevinrodrigues) [inline packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts:45]: I think these sentences need fixing as it doesn't read right at the moment.

**Review by raoufswe — APPROVED** [review]

File: packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts [hunk 2]
```diff
@@ -79,14 +79,14 @@ test.describe('PieCookieBanner - Component tests', () => {
         const cookieBanner = await page.locator(componentSelector);
```
Comment (kevinrodrigues) [inline packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts:79]: Might have missed it as i commented late on your pr (sorry). These awaits can go i think.

File: packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts (hunk 1 above)
Comment (xander-marjoram) [reply packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts:45]: something like `should emit the correct event and close the cookie banner when "Accept all" is clicked`

Comment (jamieomaguire) [reply packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts:45]: lol sorry thats awful isn't it - let me re-word

**Review by kevinrodrigues — APPROVED** [review]

**Review by raoufswe — APPROVED** [review]

**Pull request merged**